apps/src/geographyConstants.js

```diff
@@ -0,0 +1,303 @@
+export const STATES = [
```

Comment:
@islemaster I don't know if this is best place for these. You mentioned sharedConstants, but these constants are actually slightly different than the ones used for the current school dropdowns because I don't need the abbreviates/country codes.

Reply:
Hmm... it's still duplicating information in our source code though.  It would be preferable to derive this list from the one we have in Ruby (I think the `get_all_states_with_dc` helper does exactly that) and make whatever other small changes, like the blank entry at the top, that we need for the dropdown.